header.html

```diff
@@ -30,790 +44,1579 @@
+                        Business Opportunities
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/innovation">
+                      <div>
+                        Innovation Proposals
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/engineering/design-standards-and-guidelines">
+                      <div>
+                        Engineering Design Standards
+                      </div>
+                    </a>
+                  </nav>
+                  <div class="m-menu__section-heading">
+                    Our Work
                   </div>
+                  <nav class="m-menu-desktop__submenu-section" aria-label="Our Work">
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/sustainability">
+                      <div>
+                        Sustainability
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/projects/building-better-t-2020">
+                      <div>
+                        Building a Better T
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/projects/capital-transformation-programs">
+                      <div>
+                        Capital Transformation
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/projects/commuter-rail-positive-train-control-ptc">
+                      <div>
+                        Commuter Rail Positive Train Control
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/projects/better-bus-project">
+                      <div>
+                        Better Bus Project
+                      </div>
+                    </a>
+                    <a rel="noopener" class="m-menu__link" href="https://www.mbta.com/projects">
+                      <div>
+                        All MBTA Improvement Projects
+                      </div>
+                    </a>
+                  </nav>
                 </div>
               </div>
             </div>
           </div>
-          <div class="nav-item">
-            <a rel="noopener" aria-controls="contactUs" class="desktop-nav-link js-header-link collapsed" data-parent="#desktop-menu" data-target="#contactUs" href="https://www.mbta.com/customer-support" role="tab">
-              <div class="nav-link-content js-header-link__content">
-                <span class="nav-link-name">
-                  Contact Us
+        </div>
+        <div class="m-menu__language">
+        </div>
+      </nav>
+  </div>
+  <nav class="m-menu--mobile" id="navmenu">
```

Comment:
Let's either try to make the mobile menu work (when clicking the "menu" button) or delete this menu entirely. Leaning toward deleting this nav